docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3.1"
+
+services:
+  php:
+    image: php:7.1.7-cli
```

Comment:
No, I think it's a good practice to explicitly set the version. In the future something may not work with a new PHP version and it would break without notice.